src/scaffoldmaker/utils/cylindermesh.py

```diff
@@ -98,23 +98,51 @@ class CylinderCentralPath:
     Stores ellipses parameters a long the central path.
     """
 
-    def __init__(self, region, centralPath, elementsCount):
+    def __init__(self, region, centralPath, elementsCount, cylinderScaffold=True, annotationGroup=None):
```

Comment:
1. cylinderScaffold allows me to read and store the parameters in the right format without calling it via 2D path (if cylinderScaffold is False).
2. annotationGroup allows me to annotate the elements in the 2D path. I think it will be used full in the future :slightly_smiling_face:

I fixed those bugs, now it works with whole body mesh and brainstem.